tools/dds/dds-config/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ file( GLOB_RECURSE RS_DDS_CONFIG_SOURCE_FILES
     LIST_DIRECTORIES false
     RELATIVE ${PROJECT_SOURCE_DIR}
     "${CMAKE_CURRENT_LIST_DIR}/*"
+    "${PROJECT_SOURCE_DIR}/../../../third-party/rsutils/include/rsutils/type/*"
```

Comment:
This path seems weird..
relative to the source dir we only need `third-party/rsutils/include/rsutils/type/*` no?

Reply:
The source dir for the dds tool is librealsense\tools\dds\dds-config so only using part of the path won't work but I can set the THIRD_PARTY_DIR so it looks more readable:
set(THIRD_PARTY_DIR "${PROJECT_SOURCE_DIR}/../../../third-party")
"${THIRD_PARTY_DIR}/rsutils/include/rsutils/type/*"